src/linker/implementation.py

```diff
@@ -48,6 +55,21 @@ def validate(self) -> List[Optional[str]]:
     # Helper methods #
     ##################
 
+    def _format_config(self, config: Optional[Dict[str, Any]]) -> Optional[Dict[str, str]]:
+        return self._stringify_keys_values(config) if config else None
+
+    def _stringify_keys_values(self, config: Dict[str, Any]) -> Dict[str, str]:
```

Comment:
```suggestion
    def _stringify_keys_values(self, config: Union[Any, Dict[str, Any]]) -> Dict[str, str]:
```

or just Any.

Reply:
Yeah I stared at that for a while and had a serious internal struggle with whether the union of Any and anything else is informative or not. I'll change it to Any

Reply:
Wait...I was confused. I don't think at this point it can be Any - it can only be a dict b/c it only enters it `if isinstance(config, Dict)`